examples/cifar10/cifar10-splitnn/.gitignore

```diff
@@ -0,0 +1,23 @@
+# ide
```

Comment:
just a note, we could probably unify all these ignore files.
To add just one `.gitignore` say at the root of "cifar10" example folder.

Reply:
Good point. Moved it to root level of the example.